CHANGELOG.md

```diff
@@ -1,12 +1,12 @@
 For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
 
-2.1.0 (in progress)
+2.0.1 (in progress)
```

Comment:
@syerrapragada @makarandp0 

Since we already announced Edge support with 2.0.0, the changes in the next version of the SDK will not be as significant. So, I'm converting this into a patch release.